ndm-operator.yaml

```diff
@@ -180,6 +178,9 @@ spec:
         hostPath:
           path: /proc
           type: Directory
+      - name: basepath
```

Comment:
this has to be env isn't it, so that, helm variable can be added? cc: @prateekpandey14

Reply:
ENV not required, we can just simply read from the `values.yaml` file.
We have to make sure that, we should not be having multiple naming paths, to avoid the confusion.